src/hooks/useServiceContainer.ts

```diff
@@ -146,3 +141,17 @@ export const useServiceContainer = (service: Function, loadingStateDelayMs: numb
     run: useCallback(serviceContainerRunner, [service, loadingStateDelayMs]),
   };
 };
+
+const getErrorMessage = (error: Error | AxiosError): string => {
+  if (isAxiosError(error)) {
+    if (error.code === 'ERR_NETWORK') {
+      return 'Action was not successful due to the network error. Please, try again.';
+    } else if (error.response?.status === 401) {
+      return 'Action was not successful, please login first and try again. [HTTP 401 Unauthorized]';
+    }
+
+    return error.response?.data?.errorMessage ? error.response.data.errorMessage : error.toString();
```

Comment:
Just a question to consider for the future - what if we need to adjust this message to provide some more specific information from the UI side.

Next, can we provide some general prefix like `Actions was no successful.` ? At this point we are able to say http request resulted in error which means action should be always failed.

Reply:
I think `Actions was no successful` is not all that much needed and the top-level error message is obvious enough.
Additional parsing, I think, might be considered in the future if such need arises.

Reply:
Agreed